content/go/concepts/math-functions/terms/tan/tan.md

```diff
@@ -65,13 +65,8 @@ import (
 )
 
 func main() {
-  positiveInfinity := math.Inf(1)
-  negativeInfinity := math.Inf(-1)
-
-  tanPositiveInf := math.Tan(positiveInfinity)
-  tanNegativeInf := math.Tan(negativeInfinity)
-
-  fmt.Printf("The tangent of positive infinity: %f\n", tanPositiveInf)
-  fmt.Printf("The tangent of negative infinity: %f\n", tanNegativeInf)
+	fmt.Println(math.Tan(math.Inf(+1))) // Outputs: NaN
+	fmt.Println(math.Tan(math.Inf(-1))) // Outputs: NaN
+	fmt.Println(math.Tan(math.NaN()))   // Outputs: NaN
```

Comment:
```suggestion
	fmt.Println(math.Tan(math.Inf(+1)))
	fmt.Println(math.Tan(math.Inf(-1)))
	fmt.Println(math.Tan(math.NaN()))
```
We don't need the output comment on a codebyte

Reply:
Very true!! I had just fixed it and removed the unnecessary comments, and committed the adjusted branch.
Thanks for the advice!